gm4_better_armor_stands/data/better_armor_stands/functions/main.mcfunction

```diff
@@ -0,0 +1 @@
+execute as @e[type=item,nbt={Item:{id:"minecraft:writable_book"}}] at @s if entity @e[type=armor_stand,distance=..1,limit=1,sort=nearest] run function #better_armor_stands:update
```

Comment:
I don't see a function called `#better_armor_stands:update`, is this an old name for the function `apply_book`?

Reply:
Yes it is, oops